client/state/action-log/README.md

```diff
@@ -0,0 +1,38 @@
+# Action Logger
+**A Redux [store enhancer](https://github.com/reactjs/redux/blob/master/docs/Glossary.md#store-enhancer)**
+
+The Action Logger is a Redux store enhancer which records the most recently dispatched redux action
+so they can be inspected in the development environment for debugging and interactive design.
+
+## Functionality
+
+### Action logging
+
+Like the [Redux DevTools](https://github.com/zalmoxisus/redux-devtools-extension), the console dispatcher can also track a history of the actions flowing through the store.
+This can be toggled and the size of the history log can be adjusted.
+
+The following examples represent commands and code actually run inside the browser console.
+These are not snippets of bundles JavaScript inside of Calypso.
+
+```js
+// start recording actions
+// enabled by default in development
```

Comment:
`in development` looks too broad, maybe alter it to `in the development environment`?